src/binder/expr.rs

```diff
@@ -142,6 +142,12 @@ impl Binder {
             And => Node::And([l, r]),
             Or => Node::Or([l, r]),
             Xor => Node::Xor([l, r]),
+            Custom(name) => match name.as_str() {
+                "<->" => Node::VectorL2Distance([l, r]),
+                "<#>" => Node::VectorCosineDistance([l, r]),
+                "<*>" => Node::VectorDotProduct([l, r]),
```

Comment:
Is there any standard for these operators?

It does not seem to be same as pgvector:
> Also supports inner product (<#>), cosine distance (<=>)

Reply:
ahh, I used AI autocompletion, so it might be wrong. will fix.